docs/reference/cluster_manifest.md

```diff
@@ -53,8 +53,7 @@ Those parameters are grouped under the `metadata` top-level key.
 These parameters are grouped directly under  the `spec` key in the manifest.
 
 * **teamId**
-  name of the team the cluster belongs to. Changing it after the cluster
-  creation is not supported. Required field.
+  name of the team the cluster belongs to. Required field.
```

Comment:
Should inform about settings and implications. it may still be impossible.

Reply:
it's now possible to change it later. When `enable_team_id_clustername_prefix` is enabled you will only get an error on sync that cluster name does not match the { TEAM-ID }-{... format with status being set to inavlid.